pact-lang-api.js

```diff
@@ -823,6 +823,689 @@ const sendSigned = async function (signedCmd, apiHost) {
   return tx;
 }
 
+
+/**
+ * Functions for querying the Kadena Chainweb blockchain API
+ *
+ * @author Lars Kuhtz
+ * @module chainweb
```

Comment:
I think, those are module level comment, but here the code got copied into a different module

Reply:
Would it make sense for better maintainability to add it as a separate module to the project? And maybe re-export in pact-lang-api.js

Reply:
I imported `chainweb.js` as a dependency into `pact-lang-api.js` and then re-exported the chainweb.js endpoint functions we care about. So there's no need to copy over code from chainweb.js into this repo for now.